### ISSUE TITLE
Add MSVC detail info to README?

### ISSUE THREAD

**Issue by tian-lan** [brain-products/LSL-actiCHamp#2]
Had MSVC 2015, 2017 and 2019 installed. Running the connector led to error messages: "**The code execution cannot proceed because MSVCP110.dll and MSVCR110.dll were not found. Reinstalling the program may fix this problem**" 

MSVC110 suggests that VS 2012 is needed, and the connector runs fine after 2012 is installed.

This has been resolved, so instead of an issue, it's more like a request to add this info to README and let people know what error they should expect when certain packages are missing.

Cheers.

**Comment by dmedine** [comment]
Sorry you had this difficulty. Which version were you using? The latest build (https://github.com/brain-products/LSL-actiCHamp/releases/tag/1.13.1) should be using VC 142. I will make a new build tomorrow using the new CI integration. I set this up a few weeks ago, but I haven't made a new release since the source code hasn't changed since the May 8 release.

Is it possible that you have an older version of liblsl set on the PATH environment variable? This could be the root of the runtime error. Just a guess.

**Comment by dmedine** [comment]
Scratch that. What I said was wrong. I did push a v1.13.2 tag, but somehow it didn't go into the releases page. I will investigate this.

**Comment by dmedine** [comment]
Aha. I see what happened. I made a new tag for testing the CI, but it didn't make it through the merge. I will re-tag and re-push and there should be a brand-new v1.13.2 release ready to go. 

But that is not your question/issue.

I determined why this is happening, however. I had forgotten that the back-end library for actiCHamp needs this older vc runtime library. However, if you installed Recorder on your machine (which may, of course, not be the case), this should already have been installed along with the actiCHamp drivers.  

I will add a not about this in the README file.
